contrib/fossil/template.py

```diff
@@ -0,0 +1,20 @@
+pkgname = "fossil"
+pkgver = "2.23"
+pkgrel = 0
+build_style = "configure"
+configure_args = ["--prefix=/usr"]
+make_check_target = "test"
+# tests require $USER being set
+make_check_env = {"USER": "check"}
+makedepends = ["openssl-devel", "zlib-devel"]
+checkdepends = ["tcl"]
+pkgdesc = "Distributed software configuration management system"
+maintainer = "Erica Z <zerica@callcc.eu>"
+license = "BSD-2-Clause"
+url = "https://fossil-scm.org"
+source = f"https://fossil-scm.org/home/tarball/47362306a7dd7c6fc3cab77cebe5d25469b0a9448479d9718eb5c49c8337b29/fossil-src-{pkgver}.tar.gz"
```

Comment:
i think there's a `"https://www.fossil-scm.org/home/tarball/version-{pkgver}/fossil-src-{pkgver}.tar.gz` to not have a hash in it